extensions/notebook/src/book/bookTocManager.ts

```diff
@@ -221,14 +221,23 @@ export class BookTocManager implements IBookTocManager {
 	/**
 	 * Follows the same logic as the JupyterBooksCreate.ipynb. It receives a path that contains a notebooks and
 	 * a path where it creates the book. It copies the contents from one folder to another and creates a table of contents.
-	 * @param bookContentPath The path to the book folder, the basename of the path is the name of the book
-	 * @param contentFolder The path to the folder that contains the notebooks and markdown files to be added to the created book.
+	 * @param bookContentPath - The path to the book folder, the basename of the path is the name of the book
+	 * @param contentFolder - (Optional) The path to the folder that contains the notebooks and markdown files to be added to the created book.
+	 * If it's undefined then a blank notebook is attached to the book.
+	 *
 	*/
-	public async createBook(bookContentPath: string, contentFolder: string): Promise<void> {
-		await fs.promises.mkdir(bookContentPath);
-		await fs.copy(contentFolder, bookContentPath);
-		let filesinDir = await fs.readdir(bookContentPath);
-		this.tableofContents = await this.getAllFiles([], bookContentPath, filesinDir, bookContentPath);
+	public async createBook(bookContentPath: string, contentFolder?: string): Promise<void> {
+		let filesinDir: string[];
+		await fs.promises.mkdir(bookContentPath, { recursive: true });
+		if (!!contentFolder) {
```

Comment:
You don't need the `!!` - if the value is truthy it'll go in this block regardless